var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -26,7 +26,8 @@ class PyNumpy(PythonPackage):
         'numpy.distutils.command', 'numpy.distutils.fcompiler'
     ]
 
-    version('1.15.2', sha256='27a0d018f608a3fe34ac5e2b876f4c23c47e38295c47dd0775cc294cd2614bc1')
+    version('1.17.4', sha256='f58913e9227400f1395c7b800503ebfdb0772f1c33ff8cb4d6451c06cabdf316')
+    version('1.15.2', sha256='27a0d018f608a3fe34ac5e2b876f4c23c47e38295c47dd0775cc294cd2614bc1', Preferred=True)
```

Comment:
why prefer the old one?

Reply:
due to recompilation, I guess?

Reply:
@pramodk Is it important for a python module?